include/index_helper.cuh

```diff
@@ -1,4 +1,8 @@
 #pragma once
+#include "quda_define.h"
+#if defined (QUDA_TARGET_HIP)
+#include "hip/hip_runtime.h"
+#endif
```

Comment:
We can remove this once we abstract the blockIdx.x in this file

Reply:
OK. Still seems to contain blockIdx.x.